package.json

```diff
@@ -9,12 +9,18 @@
     "start": "node dist/server.js",
     "dev": "npx tsc && nodemon dist/server.js",
     "vercel-build": "npm run build",
-    "postinstall": "tsc"
+    "postinstall": "tsc",
+    "lint": "eslint ./src",
+    "lint:fix": "eslint . --fix",
+    "format": "prettier --write \"./src/**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
+    "pre-commit": "lint-staged",
+    "prepare": "husky install"
```

Comment:
when you run -> npx husky-init then they created this file with .husky(inside we have pre-commit file which uses npm run pre-commi)